src/common/types/objects/transactions.ts

```diff
@@ -1,3 +1,5 @@
+import { Signer } from '../../../models/common'
+
```

Comment:
The types here shouldn't be used at all (we're switching to the ones in `src/model`), so you don't need to worry about the right type being here.